README.md

```diff
@@ -77,6 +77,10 @@ It is usually in the form: `https://<CFPHOST>/<CONFERENCE>/schedule/export/sched
 
 It usually is in the form: `https://<CFPHOST>/conferences/<CONFERENCE>/schedule.xml`
 
+### Indico
+
+It usually is in the form: `https://<CFPHOST>/event/<CONFERENCE>/event.ics?detail=contributions`
```

Comment:
They only have .ics exports? :(

Reply:
I think so, or their custom json format.
See also https://github.com/indico/indico/issues/6199

Reply:
Gotcha, guess I'll see how simple it is and maybe add support for it some day if it's worth the trouble. Just really hate `.ics` files :)